### PR TITLE
ThingSetupManager returns created Thing instances

### PR DESCRIPTION
**Pull request by AlexTugarev**
Signed-off-by: Alex Tugarev <telekom@tugarev.de>

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java [hunk 1]
```diff
@@ -122,8 +130,9 @@ public void addThing(ThingUID thingUID, Configuration configuration, ThingUID br
      * @param enableChannels
      *            defines if all not 'advanced' channels should be enabled
      *            directly
+     * @return created {@link Thing} instance
```
Comment (dnobel) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java]: Could you please add "(can be null)" to the return comment.

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java [hunk 2]
```diff
@@ -81,28 +83,34 @@ public void addHomeGroup(String itemName, String label) {
...
+    public Thing addThing(ThingUID thingUID, Configuration configuration, ThingUID bridgeUID, String label) {
+        return addThing(thingUID, configuration, bridgeUID, label, new ArrayList<String>());
     }
 
     /**
      * Adds a thing and enables all channels.
      *
      * See {@link ThingSetupManager#addThing(ThingUID, Configuration, ThingUID, String, List, boolean)}
+     * 
+     * @return created {@link Thing} instance
```
Comment (dnobel) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java]: Could you please add "(can be null)" to the return comment.

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java [hunk 3]
```diff
@@ -81,28 +83,34 @@ public void addHomeGroup(String itemName, String label) {
...
+    public Thing addThing(ThingUID thingUID, Configuration configuration, ThingUID bridgeUID) {
+        return addThing(thingUID, configuration, bridgeUID, null);
     }
 
     /**
      * Adds a thing without group names, but enables all channels.
      *
      * See {@link ThingSetupManager#addThing(ThingUID, Configuration, ThingUID, String, List, boolean)}
+     * 
+     * @return created {@link Thing} instance
```
Comment (dnobel) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java]: Could you please add "(can be null)" to the return comment.

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java [hunk 4]
```diff
@@ -81,28 +83,34 @@ public void addHomeGroup(String itemName, String label) {
      * channels.
      *
      * See {@link ThingSetupManager#addThing(ThingUID, Configuration, ThingUID, String, List, boolean)}
+     * 
+     * @return created {@link Thing} instance
```
Comment (dnobel) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java]: Could you please add "(can be null)" to the return comment.

**Comment by dnobel** [comment]
Please also add yourself as an author to the class.

**Comment by AlexTugarev** [comment]
@dnobel the doc is fixed.

**Pull request merged**